common/lib/dependabot/pull_request_creator/gitlab.rb

```diff
@@ -109,7 +109,7 @@ def file_actions
             action: file_action(file),
             file_path: file.type == "symlink" ? file.symlink_target : file.path,
             content: file.content,
-            encoding: file.content_encoding
+            encoding: file.content_encoding == Dependabot::DependencyFile::ContentEncoding::BASE64 ? "base64" : "text"
```

Comment:
I chose to not pass constant directly and hardcode the string instead because technically the `BASE64` constant could be renamed in `dependabot-core` or `gitlab` api might introduce a differently named option in next version of api for example.

It would be better to have common method to return encoding option in both creator and updater class, but not sure where to place it, we don't seem to have a common module just for gitlab stuff.

Reply:
That makes sense 👍 I moved the file action mapping to client proxy and removed duplication